src/components/Navigation/components/Item/Item.tsx

```diff
@@ -230,19 +232,20 @@ export function Item({
 
   let secondaryNavigationMarkup: ReactNode = null;
 
-  if (subNavigationItems.length > 0 && showExpanded) {
```

Comment:
This is the scariest thing for me in this PR. This makes sub menus and their items render in the DOM, though hidden. I did this so that the aria-controls id will have a valid reference ID but not sure that's super necessary to see until the sub menu has actually rendered.

This broke the Nav in web the last time I tried this, for adding animations but Alex's fixes to [Collapsible](https://github.com/Shopify/polaris-react/pull/3606) has fixed that.

Anyone see any potential issues in doing this?

Reply:
> This makes sub menus and their items render in the DOM, though hidden. I did this so that the aria-controls id will have a valid reference ID but not sure that's super necessary to see until the sub menu has actually rendered.

Hmm, interesting! I'm not sure about aria-control, but certain screen readers require elements to always be in the DOM for certain attributes (e.g aria-live)

> Anyone sees any potential issues in doing this?

As long as the `Collapsible` changes roll out ok and collapsible hides the children from keyboard access I don't foresee any isssues.